later/lib/database/datasources/records_local_data_source.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:injectable/injectable.dart';
+import 'package:isar/isar.dart';
+import 'package:later/database/models/record_model.dart';
+
+import './records_data_source.dart';
+
+@LazySingleton(as: RecordsDataSource)
+class RecordsLocalDataSource extends RecordsDataSource {
+  final Isar _isar;
+
+  const RecordsLocalDataSource(this._isar);
+
+  @override
+  Future<void> create({
+    required String? url,
+    required String title,
```

Comment:
Prefer having such required things first in the list. Not big a deal, but looks nicer (you can see what is required first and then everything else).